### PR TITLE
Add ocrd_neat

### PR DESCRIPTION
**Pull request by kba**
This PR introduces two new processors, `ocrd-neat-export` and `ocrd-neat-import`.

These allow converting PAGE-XML to [neat](https://github.com/qurator-spk/neat) TSV (`ocrd-neat-export`) and neat TSV results back into PAGE-XML (`ocrd-neat-import`).

neat is an annotation tool originally developed for NER/NEL tasks, but adapted to be used for producing/correcting line-based Ground Truth for training OCR engines. It's a simple HTML+Javascript application that requires no additional libraries and can be used offline in any browser.

A typical workflow for GT creation with neat:

1. `ocrd-eynollah-segment` or any other [segmentation processor](https://ocr-d.de/en/workflows#layout-analysis) for the line segmentation
2. Optionally: [`ocrd-*-recognize`](https://ocr-d.de/en/workflows#text-recognition) to do an initial OCR so GT doesn't have to be written from scratch
3. `ocrd-neat-export -I PAGE_FOLDER -O TSV_FOLDER` converts every PAGE-XML to a TSV file with the transcription and coordinates of the line
4. Open [neat.html](https://github.com/qurator-spk/neat) in a browser, load TSV files from `TSV_FILES` and correct/transcribe, save (overwrite) TSV files
5. `ocrd-neat-import -I PAGE_FOLDER,TSV_FOLDER -O GT_FOLDER` will reconstitute PAGE-XML in GT_FOLDER with the updated GT
6. Extract the lines, e.g. with [`ocrd-segment-extract-lines`](https://github.com/OCR-D/ocrd_segment) and train OCR engines

### CODE REVIEW

**Review by bertsky — APPROVED** [review]
Neat!

File: .gitmodules [hunk 1]
```diff
@@ -88,3 +88,6 @@
 [submodule "ocrd_doxa"]
 	path = ocrd_doxa
 	url = https://github.com/bertsky/ocrd_doxa.git
+[submodule "ocrd_neat"]
+	path = ocrd_neat
+	url = https://github.com/qurator-spk/page2tsv
```
Comment (stweil) [inline .gitmodules:93]:
```suggestion
	url = https://github.com/qurator-spk/page2tsv.git
```

File: Makefile [hunk 2]
```diff
@@ -814,7 +828,7 @@ docker-%-git: PIP_OPTIONS = -e
 # Minimum-size selection: use Ocropy binarization, use Tesseract from PPA
 docker-mini%: DOCKER_MODULES = core ocrd_cis ocrd_fileformat ocrd_im6convert ocrd_pagetopdf ocrd_repair_inconsistencies ocrd_tesserocr ocrd_wrap tesserocr workflow-configuration ocrd_olahd_client
 # Medium-size selection: add Olena binarization and Calamari, use Tesseract from git, add evaluation
-docker-medi%: DOCKER_MODULES = core cor-asv-ann dinglehopper format-converters ocrd_calamari ocrd_cis ocrd_fileformat ocrd_im6convert ocrd_keraslm ocrd_olena ocrd_pagetopdf ocrd_repair_inconsistencies ocrd_segment ocrd_tesserocr ocrd_wrap tesseract tesserocr workflow-configuration ocrd_olahd_client
+docker-medi%: DOCKER_MODULES = core cor-asv-ann dinglehopper format-converters ocrd_calamari ocrd_cis ocrd_fileformat ocrd_im6convert ocrd_keraslm ocrd_olena ocrd_pagetopdf ocrd_repair_inconsistencies ocrd_segment ocrd_tesserocr ocrd_wrap tesseract tesserocr workflow-configuration ocrd_olahd_client ocrd_neat
```
[the remark this answers is not in the document]
Comment (stweil) [reply Makefile:831]: I suggest to restore the alphabetic sorting.

File: .gitmodules (hunk 1 above)
Comment (stweil) [reply .gitmodules:93]: Unrelated: `.git` is also missing for `eynollah`.

**Review by stweil — CHANGES_REQUESTED** [review]
I think with the two suggested changes this pull request is ready for merging.

**Pull request merged**